### DISCUSSION TITLE
Import kml labels?

### DISCUSSION THREAD

**Discussion by Mapbeek** [OpenOrienteering/mapper#2236] [Questions & Answers]
I have created a list of  feature locations in Google Earth (pushpins) and have labelled each of them.

I can import this file to mapper, the locations come in fine but the text labels do not.  

Is there a way of getting the labels to import as well?

thanks!

**Comment by dg0yt** [comment]
I can't test this right now, but: Isn't the label text captured in object tags (View \> Tag Editor)?

**Reply by Mapbeek** [reply]
Apparently not.  When I view>  tag editor  I get  a new window with columns headed 'key' and 'value' under my template window but the columns are empty.

**Reply by Mapbeek** [reply]
however the text label data must be there somewhere as I can use 'find objects' and find the pushpin objects using their label attribute.

**Reply by dg0yt** [reply]
Then it is in the tags. Did you select the object of interest before looking at tags?

**Reply by Mapbeek** [reply]
No -but when I do, I can see the label attribute of each pin marker (one at a time).

I re-imported the kmz file to see if I overlooked a symbol allocation at that step, but the dialogue only indicates the point symbols, not the text label.  This has the key 'name' in the tag info. 

So I guess I need to link the key to a text symbol somehow, but not sure how to do this.

Thanks for ongoing help.

**Reply by dg0yt** [reply]
IIUC The current behaviour is more or less as expected from what is implemented ATM: The input is a (KML) point object with a label property. The output is a (Mapper) point object with a label property. 

Your use case asks for new capabilities
 - to show the label next to a point object, or at least
 - to create a text object for the point object's label during import.

**Reply by Mapbeek** [reply]
Thanks, yes you understand what I am looking for exactly.  I thought there may be a way of doing this but did not realise it involves developing new capabilities.  

Thanks again for looking into this.

Rob
